### PR TITLE
🐤 Chip Stage 7

### PR DESCRIPTION
**Pull request by andresin87**
**TASK**: 

### Description, Motivation and Context

- Fix design with bug
- Apply stage 6 feedback
- Sort stories applying guidelines
- Add keyboard interaction definitions
- Change onClear API to get aligned with what is made in the input package

### Types of changes

- [x] ✨ New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: packages/components/chip/package.json [hunk 1]
```diff
@@ -28,8 +28,10 @@
...
   "license": "MIT",
   "dependencies": {
-    "@spark-ui/icon": "latest",
-    "@spark-ui/icons": "latest",
-    "@spark-ui/slot": "latest"
+    "@spark-ui/icon": "^2.0.2",
+    "@spark-ui/icons": "^1.19.5",
+    "@spark-ui/internal-utils": "^2.0.2",
+    "@spark-ui/slot": "^1.6.1",
+    "emulate-tab": "^1.2.1"
```
Comment (Powerplex) [inline packages/components/chip/package.json:35]: I just ping @kikoruiz :) I am a bit lost with when to use `latest` or not. Has something changed regarding this ?

File: packages/components/chip/src/Chip.doc.mdx [hunk 2]
```diff
@@ -50,44 +65,81 @@ import { Chip } from '@spark-ui/chip'
...
+<Canvas of={stories.Disabled} />
 
-## Assist
+### Max Width
+
+The max width of any Chip is 240px including its borders, icons, and clear button.
+
+<Canvas of={stories.MaxWidth} />
+
+## Accessibility
```
Comment (Powerplex) [inline packages/components/chip/src/Chip.doc.mdx:109]: a11y section should be moved to the end of the file :)

File: packages/components/chip/package.json (hunk 1 above)
Comment (andresin87) [reply packages/components/chip/package.json:35]: I realise `latest` only works in lerna CI publish step, but locally it does not infer to the current changed package (linking ints changes) and downloads the latest published

**Review by turolopezsanabria — COMMENTED** [review]
👍🏼

**Pull request merged**